NanoGardener/python/modules/qq2vvEWKcorrectionsWeightProducer.py

```diff
@@ -235,8 +459,8 @@ def analyze(self, event):
           temp_etaq2 = -1
           temp_phiq1 = -1
           temp_phiq2 = -1
-          temp_idlq1 = -1
-          temp_idlq2 = -1
+          temp_idq1 = -1
```

Comment:
ah ... and this I think was a bug left there before ... although it had no effect ... python magic assignment